manifest.json

```diff
@@ -22,15 +22,23 @@
       "scripts/background.js"
     ]
   },
-  "browser_action": {
+  "page_action": {
+    "show_matches": ["*://*/*"],
+    "hide_matches": [
+      "*://www.instapaper.com/u*",
+      "*://www.instapaper.com/read/*",
+      "*://www.instapaper.com/api/*",
+      "*://www.instapaper.com/text?u=*"
+    ],
     "default_popup": "browserAction.html",
+    "default_title": "Read with Instapaper",
     "browser_style": true,
     "default_icon": {
       "16": "icons/icon.svg",
       "19": "icons/icon.svg",
       "24": "icons/icon.svg",
       "32": "icons/icon.svg",
-      "38": "icons/icon.svg"
+      "64": "icons/icon.svg"
```

Comment:
This was actually supposed to be 38 (2x pixel density version of 19x19.)

Reply:
Yes, This was a test of mine when it was in `browser_action` that I forget to revert. Fixed.